### ISSUE TITLE
Make comments more readable

### ISSUE THREAD

**Issue by caio-felipee** [iagorrr/noctishc.nvim#2]
Currently, it's not that easy to read code comments, unless you have your screen brightness kind of high.
Example preview: 

![image](https://github.com/iagorrr/noctis-high-contrast.nvim/assets/45669355/51f3cb68-de92-4cf5-bd3c-51d0fe2474e2)

It is a struggle to read, as you can see on line 1 and 14.

**Comment by iagorrr** [comment]
I totally agree, i think that if we make the comment color as `#888888`, as suggest the original colors in the `README  it will be fine, i already have a color for it which is "discretGray"

**Comment by iagorrr** [comment]
I believe the changes were made by the mentioned pull request, can you confirm it ?

**Comment by iagorrr** [comment]
I've check and it looks much better now :+1:
